test/lib/interpreters/split_clustering_test.rb

```diff
@@ -315,6 +316,9 @@ def test_unavailable_days_taken_into_account_vehicle_work_day
       vrp[:services][0][:activity][:timewindows] = [{ start: 0, end: 10, day_index: 0 }]
       vrp[:services][3][:activity][:timewindows] = [{ start: 0, end: 10, day_index: 1 }]
       service_vrp = { vrp: TestHelper.create(vrp), service: :demo }
+      service_vrp[:vrp].services.each{ |s|
+        s.skills = []
+      }
```

Comment:
unless you have a specific reason?
```suggestion
      service_vrp[:vrp].services.each{ |s| s.skills = [] }
```